src/content/docs/en/basics/layouts.mdx

```diff
@@ -45,12 +45,13 @@ const { title } = Astro.props;
     </article>
     <Footer />
   </body>
-  <style>
-    h1 {
-      font-size: 2rem;
-    }
-  </style>
 </html>
+
```

Comment:
Let's NOT change this to be outside of the `<html>` tag, even though we discovered it's possible. Just because it *can* be done, doesn't mean that it should! 😅 

I think our main example should still show the most common, supported use case!

Reply:
Good idea. I've fixed it.